akka-persistence/src/test/scala/akka/persistence/fsm/PersistentFSMSpec.scala

```diff
@@ -75,7 +75,7 @@ abstract class PersistentFSMSpec(config: Config) extends PersistenceSpec(config)
       expectMsg(CurrentState(fsmRef, LookingAround, None))
       expectMsg(Transition(fsmRef, LookingAround, Shopping, Some(1 second)))
 
-      within(0.9 seconds, remainingOrDefault) {
+      within(0.5 seconds, remainingOrDefault) {
```

Comment:
Ah this is minimum/maximum, that wasn't intuitive to me - perhaps naming those parameters `within(min = 0.9 seconds, max = remainingOrDefault)` might make it a little easier to read, but OK this way as well :)

Reply:
Was thinking about that, but then thought that we only see the `Transition` message and not when it actually happened. But maybe that would be good enough to not have it fail....